src/DotVVM.Framework/Controls/FileUpload.cs

```diff
@@ -210,7 +210,7 @@ private void RenderUploadButton(IHtmlWriter writer)
             writer.AddAttribute("class", "dotvvm-upload-button");
             writer.AddKnockoutDataBind("visible", "!IsBusy()");
             writer.RenderBeginTag("span");
-            writer.AddAttribute("href", "#");
+            writer.AddAttribute("href", "c");
```

Comment:
Is the "c" intentional, or is it a typo?

Reply:
typo
It should be fixed now